server/fleet/microsoft_mdm.go

```diff
@@ -361,10 +396,11 @@ type RequestFilter struct {
 /// https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-mde2/6ba9c509-8bce-4899-85b2-8c3d41f8f845
 
 type RequestSecurityToken struct {
-	TokenType           string              `xml:"TokenType"`
-	RequestType         string              `xml:"RequestType"`
-	BinarySecurityToken BinarySecurityToken `xml:"BinarySecurityToken"`
-	AdditionalContext   AdditionalContext   `xml:"AdditionalContext"`
+	TokenType           string                  `xml:"TokenType"`
+	RequestType         string                  `xml:"RequestType"`
+	BinarySecurityToken BinarySecurityToken     `xml:"BinarySecurityToken"`
+	AdditionalContext   AdditionalContext       `xml:"AdditionalContext"`
+	MapContextItems     *map[string]ContextItem `xml:"-"`
```

Comment:
Unless this is meant to be used elsewhere, currently this is only used inside the `server/fleet` package, should be unexported. Also, no need to have it be a pointer to a map (it might've been for special-cases during xml encoding/decoding, but it is explicitly ignored with the xml:"-" struct tag so that's not it), a map is nil by default so you can still detect if it has been filled or not without it being a pointer to a map.

Reply:
yes, this type is used on the XML WAP provisioning logic being done on the enrollment endpoint. I'll leave it exported. Thanks for the tip on map pointer. It is currently not used for XML parsing, so I've just fixed this.